CMakeLists.txt

```diff
@@ -18,6 +19,14 @@ option(BENCHMARK_USE_LIBCXX "Build and test using libc++ as the standard library
 option(BENCHMARK_BUILD_32_BITS "Build a 32 bit version of the library." OFF)
 option(BENCHMARK_ENABLE_INSTALL "Enable installation of benchmark. (Projects embedding benchmark may want to turn this OFF.)" ON)
 
+# This option is useful when building Benchmark in odd configurations, such as 32bit mode
+# or against libc++.
+option(BENCHMARK_BUILD_GTEST_INTREE "Download and build a custom copy of GTest to use" ON)
```

Comment:
This should be two options:
1. build gtest
2. if build gtest, is downloading allowed? this really really should default to false. build systems downloading stuff from internet is such a disgrace :/

Reply:
SGTM. Thanks for the feedback.